spec/computer_player_spec.rb

```diff
@@ -20,4 +22,23 @@
       expect(computer_player.assign_marker_as('X')).to eq('X')
     end
   end
+
+  describe '#make_unbeatable' do
```

Comment:
i think a good test case you might be missing: what would happen if two unbeatable computers were to play one another?

Reply:
Agreed. Added to the list!